### ISSUE TITLE
Another small suggestion for the Build and Install Guide wiki page

### ISSUE THREAD

**Issue by paddle-to-the-sea** [trunkmaster/nextspace#130]
Re: "3. Build NEXTSPACE: 5. Take this step if you want GNUstep libraries with fresh patches:...Build and install GNUstep Back:"

When I tried to run `$ ./configure --enable-server=x11 --enable-graphics=art --with-name=art` I got the error mentioned by @markusmeyerhofer: `configure: error: You must have the gnustep-make package installed and set up the GNUSTEP_MAKEFILES environment variable to contain the path to the makefiles directory before configuring!`

I logged out and logged back in, and then `$ ./configure --enable-server=x11 --enable-graphics=art --with-name=art` ran just fine. I think logging out and back in reloaded the correct environment variables.

Here's the reference from an earlier issue:
Thanks so far. I reinstalled from scratch and again followed the wiki verbatim, though still stuck at the configure step:

_configure: error: You must have the gnustep-make package installed and set up the GNUSTEP_MAKEFILES environment variable to contain the path to the makefiles directory before configuring!_

Installing the gnustep-make package, however, fails due to conflicting packages:
_Error: nextspace-gnustep conflicts with gnustep-filesystem-2.7.0-1.el7.x86_64_

So the install error mentioned in my original post stems from gnustep-make conflicting. Any idea how to circumvent the mentioned configure error? Maybe setting GNUSTEP_MAKEFILES environment variables somehow?

Next, I skipped step 3.5 and continued. In step 7 make fails:
 _make: *** No rule to make target `/aggregate.make'.  Stop._

I stopped here due to obvious reasons :-)

_Originally posted by @markusmeyerhofer in https://github.com/trunkmaster/nextspace/issues/30#issuecomment-422583997_

**Comment by uchuugaka** [comment]
You do not necessarily need to log out and log in again to affect environment variables.
In Bash, you can reload one of the various Bash configuration files by using
`source ~/.profile` or `source path/to/file` as it could be a .profile or .bashrc or anything you like, loaded by one of those. Each shell loads its config files in a particular order and some might not be loaded based on session types (SSH or GUI and so on).
**source** can also just be a dot in Bash, so these are also valid:
`. path/to/file`

**Comment by trunkmaster** [comment]
@paddle-to-the-sea, do you have installed `nextspace-core` and `nextspace-core-devel` packages?
The error you've got means you have no installed `/etc/profile.d/nextspace.sh` file that includes all necessary environment variables defined. `nextspace-core-devel` contains appropriate `gnustep-make` files. You don't need to install neither `gnustep-make` nor `gnustep-filesystem` packages.

**Comment by paddle-to-the-sea** [comment]
> @paddle-to-the-sea, do you have installed `nextspace-core` and `nextspace-core-devel` packages?

@trunkmaster, yes I have those installed. I think `/etc/profile.d/nextspace.sh` wasn't sourced properly until I logged out and back in again. Of course @uchuugaka's advice above is the more savvy path than logging out and in again.

**Comment by trunkmaster** [comment]
@paddle-to-the-sea, of course files from `/etc/profile.d` are read at user logon. I think wiki guide is correct because until step 3.3 all actions are executed as `root`. What do you think?

**Comment by paddle-to-the-sea** [comment]
@trunkmaster Yes, that sounds right. I think I was using sudo for many of the steps from my already established regular/admin user account, so my sequence of logins didn't precisely match what the instructions have.

In the near future, I might start over from scratch and pay closer attention to the order of logging in and when the environmental variables are set, but in any event I'll close this for now.

Thanks!
